careers/README.md

```diff
@@ -0,0 +1,27 @@
+# DVELP Careers
+
+We're looking to expand our team of talented people, and want to be as open about the process and opportunities as possible - hence our having all our careers information on GitHub.
+
+## Open Opportunities
+
+We're currently looking to recruit the following roles:
+
+- [Senior Fullstack Node Engineer](jobs/senior-fullstack-node-engineer.md)
+
+## Hiring Process
+
+### Apply
+
+If you want to apply for any of the positions, please email [careers@dvelp.co.uk](mailto:careers@dvelp.co.uk) with your CV.
+
+### Interviews
+
+Once your application has been reviewed, there will then be a series of interviews to make sure we're the right fit for you, and vice versa. These are done one-at-a-time, with your progressing to the next if successful in the previous. We will try to keep these as close together as possible, and work with you to find appropriate times, be those in or out of working hours - we know it's not easy to get time off during the day for this!
```

Comment:
`there will then be a series of interviews` -> `there will be a series of interviews`

`with your progressing to the next if successful in the previous` -> `with you progressing to the next if successful in the previous`